aten/src/ATen/native/transformers/transformer.cpp

```diff
@@ -128,6 +127,21 @@ Tensor transformer_encoder_layer_forward(
 #if BETTER_TRANSFORMER_USE_FLASH_ATTENTION
   }
 #endif
+  x = std::get<0>(at::_native_multi_head_attention(
```

Comment:
Commenting just in case you accidental missed this

Reply:
I think this needs to be here, before the avoided dispatch but the native_multiheaded_attention worked for all tensors subclasses so now using the dispatcher to correctly dispatch to cpu or cuda